### ISSUE TITLE
Potential bug in the analysis of the correlation matrix of CMA-ES

### ISSUE THREAD

**Issue by mlazzarin** [N3PDF/mcnntunes#9]
In the minimization step, after the execution of the CMA-ES algorithm, the covariance matrix is decomposed in eigenvalues and eigenvectors, and then a 1-sigma eigenvector basis is computed by adding the eigenvectors times the square root of the eigenvalues to the best parameters.

https://github.com/scarrazza/mcnntunes/blob/16c73bda6fdcc3d72eac19f54671ab521d703228/src/mcnntunelib/app.py#L312-L327

I've noticed that `cov` is computed with the unscaled std `best_std`:
https://github.com/scarrazza/mcnntunes/blob/16c73bda6fdcc3d72eac19f54671ab521d703228/src/mcnntunelib/app.py#L298-L300
while the 1-sigma eigenvector basis is computed by using `result[0]`, which I think is scaled.
Then, we unscale the basis (line 327), so `result[0]` gets mapped to `best_x`. However, the contribution of the covariance matrix is unscaled twice, isn't it?

In other words, I think that we should replace `result[0]` with `best_x` in line 324 and remove the unscale function call in line 327.

@scarrazza could you take a look and see whether I'm right or I didn't understood these lines?  

All of this is in the master branch.
